staging_service/app.py

```diff
@@ -106,6 +118,44 @@ async def bulk_specification(request: web.Request) -> web.json_response:
     return web.HTTPInternalServerError(text=errtext, content_type=_APP_JSON)
 
 
+@routes.post("/write_bulk_specification/")
+async def write_bulk_specification(request: web.Request) -> web.json_response:
```

Comment:
Should add this to the readme

Reply:
Needs a docstring

Reply:
> Should add this to the readme
That's in the next PR, I didn't want to make this PR too big

Reply:
> Needs a docstring

Wow, sloppy. Fixed

Reply:
> Might need to check if this needs to be added to [the narrative javascript staging service client]

I'm not nearly good enough at javascript to do anything about it, so my plan is to let the person who first uses this endpoint decide how they want to implement the narrative side of things.